src/components/Input/index.jsx

```diff
@@ -0,0 +1,62 @@
+import cls from 'classnames';
+import {
+  flex, margin, borders, colors, typography, padding,
+}
+  from '../../theme';
+
+import styles from './styles.module.css';
+
+export default function Input({ setTextInput, inputLabel, darkMode }) {
+  const typingHandler = (event) => {
+    if (event.target.value !== '') {
+      setTextInput(event.target.value);
+    } else {
+      setTextInput('');
```

Comment:
Not sure what this if is for. If the value is '', and you set it to ''. Then that was the same as setting it to the value.